packages/config/config/devices/0x0345/templates/swidget_template.json

```
@@ -0,0 +1,1504 @@
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 65535,
+		"unsigned": true,
+		"defaultValue": 3600
+	},
+	"air_quality_alert_entry_set_value": {
+		// param 72
+		"label": "Air Quality Alert Entry SET Value",
+		"description": "BASIC SET value sent to associated nodes on entry into air quality alert state.",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"unsigned": true,
+		"defaultValue": 255
+	},
+	"air_quality_alert_exit_set_value": {
+		// param 73
+		"label": "Air Quality Alert Exit SET Value",
+		"description": "BASIC SET value sent to associated nodes on exit from air quality alert state.",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"unsigned": true,
+		"defaultValue": 0
+	},
+	"air_quality_actions_control_associations_on_entry": {
+		// param 74b0
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Air Quality Actions: CONTROL ASSOCIATIONS ON ENTRY",
+		"description": "If set, send BASIC SET with 'Air Quality Alert Entry SET Value' to 'Air Quality' association group on entry into air quality alert state."
+	},
+	"air_quality_actions_control_associations_on_exit": {
+		// param 74b1
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Air Quality Actions: CONTROL ASSOCIATIONS ON EXIT",
+		"description": "If set, send BASIC SET with 'Air Quality Alert Exit SET Value' to 'Air Quality' association group on entry into air quality alert state."
+	},
+	"air_quality_actions_control_host_on_entry": {
+		// param 74b2
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Air Quality Actions: CONTROL HOST ON ENTRY",
+		"description": "If set, execute BASIC SET with 'Air Quality Alert Entry SET Value' to 'Air Quality' association group on entry into air quality alert state."
+	},
+	"air_quality_actions_host_on_exit": {
+		// param 74b3
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Air Quality Actions: CONTROL HOST ON EXIT",
+		"description": "If set, execute BASIC SET with 'Air Quality Alert Exit SET Value' to 'Air Quality' association group on entry into air quality alert state."
+	},
+	"air_quality_actions_notify_transitions": {
+		// param 74b4
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "TAir Quality Actions: NOTIFY TRANSITIONS",
+		"description": "If set, send NOTIFICATION REPORT (Home Health: VOC Level Status Event) to Lifeline with updated pollution level value on change of air quality state and during the renotification period while in air quality alert state (if enabled)."
+	},
+	"temperature_high_entry_level": {
+		// param 80
+		"label": "Temperature High Entry Level",
+		"description": "Used to set the temperature value that indicates entry into a high/over temperature state.",
+		"unit": "Tenths of a Degree",
```

```suggestion
		"unit": "0.1 °C",
```
or is it Fahrenheit?

---

It can be either. Depending on if it's set to measure in C or F